inc/global_assets/namespace.php

```diff
@@ -37,26 +38,25 @@ function bootstrap() {
 }
 
 /**
- * Load Asset Mnnager Framework.
+ * Load Asset Manager Framework for WordPress.
  *
  * @return void
  */
-function load_amf() {
-	$config = Altis\get_config()['modules']['media'];
+function load_amf_wp() {
+	// Wait until post installation.
+	if ( defined( 'WP_INITIAL_INSTALL' ) && WP_INITIAL_INSTALL ) {
+		return;
+	}
 
 	// Load Asset Manager Framework.
-	require_once Altis\ROOT_DIR . '/vendor/humanmade/asset-manager-framework/plugin.php';
+	Media\load_amf();
```

Comment:
What if someone is conditionally loading AMF? If both things are hooked to `plugins_loaded`, wouldn't AMF get loaded regardless? Should there be an `unload_amf` or a check to see if AMF should be loaded before actually running the function? Maybe this should be a 

```php
do_action( 'altis.media.load_amf' );
```

```php
// In the bootstrap()...
add_action( 'altis.media.load_amf', __NAMESPACE__ . '\\Media\\load_amf' );
```